flexget/plugins/plugin_rarfile.py

```diff
@@ -0,0 +1,211 @@
+from __future__ import unicode_literals, division, absolute_import
+import logging
+import os
+import re
+
+from flexget import plugin
+from flexget.entry import Entry
+from flexget.event import event
+from flexget.utils.template import render_from_entry, RenderError
+
+try:
+    import rarfile
+except ImportError:
+    pass
```

Comment:
You can just set `rarfile = None` here, then test for that at runtime instead of the current hack. :wink: